File: Easy/search_insert_position.py
class Solution(object):
    def searchInsert(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: int
        """
        index = 0
        for i, n in enumerate(nums):
            if n >= target:
                index = i
                break
        
        if len(nums) > 0 and index == 0 and nums[len(nums) - 1] < target:
            return len(nums)
        
        return index

File: Easy/search_insert_position.php
<?php

class Solution {

    /**
     * @param Integer[] $nums
     * @param Integer $target
     * @return Integer
     */
    function searchInsert($nums, $target) {
        $index = 0;
        foreach($nums as $i => $n) {
            if($n >= $target) {
                $index = $i;
                break;
            }
        }

        if (count($nums) > 0 && $index == 0 && $nums[count($nums)-1] < $target) {
            return count($nums);
        }

        return $index;
    }
}

$solution = new Solution;

print_r($solution->searchInsert(array(1, 3, 5, 6), 5));
